target/linux/octeon/image/Makefile

```diff
@@ -7,13 +7,18 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
+define Build/append-dtb-to-elf
+	$(TARGET_CROSS)objcopy --update-section .appended_dtb=$(KDIR)/image-$(DEVICE_DTS).dtb $@
+endef
+
 define Build/strip-kernel
 	# Workaround pre-SDK-1.9.0 u-boot versions not handling the .notes section
 	$(TARGET_CROSS)strip -R .notes $@ -o $@.stripped && mv $@.stripped $@
 endef
 
 define Device/Default
   PROFILES = Default $$(DEVICE_NAME)
+  DTS_DIR := $(DTS_DIR)/cavium-octeon
```

Comment:
DTS_DIR is a target variable, not a device variable. So, it should be assigned outside of Device/Default.

Reply:
would it be suitable to add DTS_DIR to <target>/Makefile ?

Reply:
or maybe we can use `DEVICE_DTS_DIR := $(DTS_DIR)/cavium-octeon` in `Device/Default` instead?
what is preferred?